### PR TITLE
[Chore] Bootstrap project using create-next-app

### PR DESCRIPTION
**Pull request by tyrro**
## What happened 👀

Resolves https://github.com/tyrro/react-survey/issues/1
Setup project using the command `npx create-next-app@latest`

## Insight 📝

N/A
 
## Proof Of Work 📹

image

### CODE REVIEW

File: public/vercel.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="283" height="64" viewBox="0 0 283 64" fill="none" 
+    xmlns="http://www.w3.org/2000/svg">
+    <path d="M141.04 16c-11.04 0-19 7.2-19 18s8.96 18 20 18c6.67 0 12.55-2.64 16.19-7.09l-7.65-4.42c-2.02 2.21-5.09 3.5-8.54 3.5-4.79 0-8.86-2.5-10.37-6.5h28.02c.22-1.12.35-2.28.35-3.5 0-10.79-7.96-17.99-19-17.99zm-9.46 14.5c1.25-3.99 4.67-6.5 9.45-6.5 4.79 0 8.21 2.51 9.45 6.5h-18.9zM248.72 16c-11.04 0-19 7.2-19 18s8.96 18 20 18c6.67 0 12.55-2.64 16.19-7.09l-7.65-4.42c-2.02 2.21-5.09 3.5-8.54 3.5-4.79 0-8.86-2.5-10.37-6.5h28.02c.22-1.12.35-2.28.35-3.5 0-10.79-7.96-17.99-19-17.99zm-9.45 14.5c1.25-3.99 4.67-6.5 9.45-6.5 4.79 0 8.21 2.51 9.45 6.5h-18.9zM200.24 34c0 6 3.92 10 10 10 4.12 0 7.21-1.87 8.8-4.92l7.68 4.43c-3.18 5.3-9.14 8.49-16.48 8.49-11.05 0-19-7.2-19-18s7.96-18 19-18c7.34 0 13.29 3.19 16.48 8.49l-7.68 4.43c-1.59-3.05-4.68-4.92-8.8-4.92-6.07 0-10 4-10 10zm82.48-29v46h-9V5h9zM36.95 0L73.9 64H0L36.95 0zm92.38 5l-27.71 48L73.91 5H84.3l17.32 30 17.32-30h10.39zm58.91 12v9.69c-1-.29-2.06-.49-3.2-.49-5.81 0-10 4-10 10V51h-9V17h9v9.2c0-5.08 5.91-9.2 13.2-9.2z" fill="#000"/>
+</svg>
```
Comment (malparty) [inline public/vercel.svg:4]:
```suggestion
</svg>

```

Should we add an ending line here? :)

Comment (tyrro) [reply public/vercel.svg:4]: absolutely 😄

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "react-survey",
+  "version": "0.1.0",
+  "private": true,
```
Comment (malparty) [inline package.json:4]: Should we add the [engines](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines) versions as well? This way it is easy to find back what node/yarn version is working with this project :) 

Or, if you want to use [asdf](https://github.com/asdf-vm/asdf) (which seems very interesting), then a `.tool-versions` file would be enough ;-)

File: README.md [hunk 3]
```diff
@@ -1,28 +1,34 @@
...
-This project is maintained and funded by Nimble.
+- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
 
-We love open source and do our part in sharing our work with the community!
-See [our other projects][community] or [hire our team][hire] to help build your product.
+You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
 
-[community]: https://github.com/nimblehq
-[hire]: https://nimblehq.co/
```
Comment (malparty) [inline README.md:28]: As this is a public repository, what about keeping the Nimble "hiring" message? :D  This way we might get to find more awesome teammates from Github ^^

**Review by malparty — CHANGES_REQUESTED** [review]
Only minor comments, feel free to challenge them of course ;-)

File: package.json (hunk 2 above)
Comment (tyrro) [reply package.json:4]: yeah, we can specify the engines versions as well, done!

File: README.md (hunk 3 above)
Comment (tyrro) [reply README.md:28]: sure!
actually the readme will go through further changes when the ci/cd will be introduced hopefully, kept the hiring message!

**Review by malparty — APPROVED** [review]
All good! 🚀

File: pages/index.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import Head from "next/head";
+import Image from "next/image";
+import styles from "../styles/Home.module.css";
```
Comment (carryall) [inline pages/index.js:3]: I'm just curious do we have any plan to add the [absolute imports or module path aliases](https://nextjs.org/docs/advanced-features/module-path-aliases)? so later when the project grow bigger and there's a lot of stylesheets to import we can use something like `styles/**` or `@/styles/**`

File: public/vercel.svg [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<svg width="283" height="64" viewBox="0 0 283 64" fill="none"
+    xmlns="http://www.w3.org/2000/svg">
+    <path d="M141.04 16c-11.04 0-19 7.2-19 18s8.96 18 20 18c6.67 0 12.55-2.64 16.19-7.09l-7.65-4.42c-2.02 2.21-5.09 3.5-8.54 3.5-4.79 0-8.86-2.5-10.37-6.5h28.02c.22-1.12.35-2.28.35-3.5 0-10.79-7.96-17.99-19-17.99zm-9.46 14.5c1.25-3.99 4.67-6.5 9.45-6.5 4.79 0 8.21 2.51 9.45 6.5h-18.9zM248.72 16c-11.04 0-19 7.2-19 18s8.96 18 20 18c6.67 0 12.55-2.64 16.19-7.09l-7.65-4.42c-2.02 2.21-5.09 3.5-8.54 3.5-4.79 0-8.86-2.5-10.37-6.5h28.02c.22-1.12.35-2.28.35-3.5 0-10.79-7.96-17.99-19-17.99zm-9.45 14.5c1.25-3.99 4.67-6.5 9.45-6.5 4.79 0 8.21 2.51 9.45 6.5h-18.9zM200.24 34c0 6 3.92 10 10 10 4.12 0 7.21-1.87 8.8-4.92l7.68 4.43c-3.18 5.3-9.14 8.49-16.48 8.49-11.05 0-19-7.2-19-18s7.96-18 19-18c7.34 0 13.29 3.19 16.48 8.49l-7.68 4.43c-1.59-3.05-4.68-4.92-8.8-4.92-6.07 0-10 4-10 10zm82.48-29v46h-9V5h9zM36.95 0L73.9 64H0L36.95 0zm92.38 5l-27.71 48L73.91 5H84.3l17.32 30 17.32-30h10.39zm58.91 12v9.69c-1-.29-2.06-.49-3.2-.49-5.81 0-10 4-10 10V51h-9V17h9v9.2c0-5.08 5.91-9.2 13.2-9.2z" fill="#000"/>
+</svg>
```
Comment (carryall) [inline public/vercel.svg:4]: Maybe we should format the SVG file with a soft tab (2 spaces) as well 🙌🏼 

```suggestion
<svg width="283" height="64" viewBox="0 0 283 64" fill="none"
  xmlns="http://www.w3.org/2000/svg">
  <path d="M141.04 16c-11.04 0-19 7.2-19 18s8.96 18 20 18c6.67 0 12.55-2.64 16.19-7.09l-7.65-4.42c-2.02 2.21-5.09 3.5-8.54 3.5-4.79 0-8.86-2.5-10.37-6.5h28.02c.22-1.12.35-2.28.35-3.5 0-10.79-7.96-17.99-19-17.99zm-9.46 14.5c1.25-3.99 4.67-6.5 9.45-6.5 4.79 0 8.21 2.51 9.45 6.5h-18.9zM248.72 16c-11.04 0-19 7.2-19 18s8.96 18 20 18c6.67 0 12.55-2.64 16.19-7.09l-7.65-4.42c-2.02 2.21-5.09 3.5-8.54 3.5-4.79 0-8.86-2.5-10.37-6.5h28.02c.22-1.12.35-2.28.35-3.5 0-10.79-7.96-17.99-19-17.99zm-9.45 14.5c1.25-3.99 4.67-6.5 9.45-6.5 4.79 0 8.21 2.51 9.45 6.5h-18.9zM200.24 34c0 6 3.92 10 10 10 4.12 0 7.21-1.87 8.8-4.92l7.68 4.43c-3.18 5.3-9.14 8.49-16.48 8.49-11.05 0-19-7.2-19-18s7.96-18 19-18c7.34 0 13.29 3.19 16.48 8.49l-7.68 4.43c-1.59-3.05-4.68-4.92-8.8-4.92-6.07 0-10 4-10 10zm82.48-29v46h-9V5h9zM36.95 0L73.9 64H0L36.95 0zm92.38 5l-27.71 48L73.91 5H84.3l17.32 30 17.32-30h10.39zm58.91 12v9.69c-1-.29-2.06-.49-3.2-.49-5.81 0-10 4-10 10V51h-9V17h9v9.2c0-5.08 5.91-9.2 13.2-9.2z" fill="#000"/>
</svg>
```

**Review by carryall — APPROVED** [review]

File: pages/index.js (hunk 4 above)
Comment (tyrro) [reply pages/index.js:3]: @carryall , I think we might add it, but IMHO we can skip it in this initialization phase for now and can look into it during https://github.com/tyrro/react-survey/issues/9 , What do you think?

File: public/vercel.svg (hunk 5 above)
Comment (tyrro) [reply public/vercel.svg:4]: nice catch, done!

File: pages/index.js (hunk 4 above)
Comment (carryall) [reply pages/index.js:3]: works for me 🙌🏼

**Pull request merged**